cloudslang_cli.rst

```diff
@@ -208,11 +208,10 @@ system properties files are being used and they contain a system
 property with the same fully qualified name, the property in the file
 that is loaded last will overwrite the others with the same name.
 
-System property files can be loaded automatically if placed in a folder
-named ``properties`` in the CLI's **bin** directory. If the
-flow or operation requires a system properties file that is not loaded
-automatically, use the ``--spf`` flag and a comma-separated list of file
-paths.
+System property files can be loaded automatically if placed in a folder located
+at ``cslang/configuration/properties``. If the flow or operation requires a
```

Comment:
for properties files the search is recursive - meaning all the SP files will be loaded recursively from this parent folder - should mention this